File: leet_code/string/58/58_1.py
class Solution:
    def lengthOfWord(seft,s:str)-> int:
        lenght = 0
        start = len(s)-1
        while start >=0 and s[start]== ' ':
            start-=1
        while start>=0 and s[start]!= ' ':
            lenght+=1
            start-=1
        return lenght  
s = Solution()
print(s.lengthOfWord('Hello World'))

File: leet_code/string/58/58_1.dart
class Solution {
  int lengthOfLastWord(String s) {
    int lenght = 0;
    int start = s.length - 1;
    while (start >= 0 && s[start] == ' ') {
      start--;
    }
    while (start >= 0 && s[start] != ' ') {
      lenght++;
      start--;
    }
    return lenght;
  }
}

void main() {
  Solution s = Solution();
  print(s.lengthOfLastWord('Hello World'));
}
